tests/unit/lms/extensions/feature_flags/_helpers_test.py

```diff
@@ -1,3 +1,5 @@
+from unittest.mock import sentinel
```

Comment:
The diff here is a bit messy but:

 * `TestAsTristate` is all new
 * `TestFeatureFlagsCookieHelper` has been written again from scratch and much shorter
 * `TestJWTCookieHelper` is unchanged